fontir/src/orchestration.rs

```diff
@@ -20,8 +23,8 @@ use crate::{error::WorkError, ir, paths::Paths, source::Input};
 #[derive(Clone, Debug, PartialEq, Eq, Hash, PartialOrd, Ord)]
 pub enum WorkIdentifier {
     StaticMetadata,
-    Glyph(String),
-    GlyphIrDelete(String),
+    Glyph(GlyphName),
+    GlyphIrDelete(),
```

Comment:
Should probably remove the parentheses for this variant.

Reply:
I'm mildly surprised that's not a clippy, when would you specifically *want* that?

Reply:
Fixed.